bot/role_management/role_management.py

```diff
@@ -131,6 +130,203 @@ async def module_role_error(self, ctx: commands.Context, error: commands.Command
+            await ctx.send(":x: Für den angegebenen Emoji existiert leider keine Reaction-Role.")
+            return
+
+        self._db_connector.remove_reaction_role(message.id, emoji)
+        await message.clear_reaction(emoji)
+        log.info("A reaction role has been removed from the message with id %s.", message.id)
+
+        if len(message.reactions) == 1:
+            self._db_connector.remove_reaction_role_group(message.id)
+
+        await ctx.send(":white_check_mark: Die Reaction-Role wurde erfolgreich entfernt.")
+
+    @reaction_role.command(name="clear")
+    @commands.is_owner()
+    @command_log
+    async def clear_reaction_roles(self, ctx: commands.Context, message: discord.Message):
+        """Command Handler for the subcommand `clear` of the `reactionrole` command.
+
+        Removes all the reaction from a specified message and deletes all the corresponding database entry of the
+        reaction roles.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            message (discord.Message): The message from which the reaction roles should be removed.
+        """
+        if message.channel.id != self.ch_role.id:
+            await ctx.send(f":information_source: Nachrichten außerhalb des Kanals {self.ch_role.mention} können keine "
+                           f"Reaction-Roles besitzen.")
+            return
+        had_reaction_roles = self._db_connector.clear_reaction_roles(message.id)
+        self._db_connector.remove_reaction_role_group(message.id)
+
+        if not had_reaction_roles:
+            await ctx.send("Die von dir angegebene Nachricht hat keine Reaction-Roles. :face_with_monocle:")
+            return
+
+        await message.clear_reactions()
+        log.info("All reaction roles of the message with id %s have been removed.", message.id)
+
+        await ctx.send(":white_check_mark: Die Reaction-Roles wurden erfolgreich entfernt.")
+
+    @reaction_role.command(name="unique")
+    @commands.is_owner()
+    @command_log
+    async def toggle_reaction_roles_exclusiveness(self, ctx: commands.Context, message: discord.Message):
+        """Command Handler for the subcommand `unique` of the `reactionrole` command.
+
+        Marks all the reaction roles of a message as "unique" by adding the message id to a specific table in the db.
+        This means that users can only have one of the configured reaction roles of this message at a time.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            message (discord.Message): The message from which the reaction roles should be removed.
+        """
+        if len(message.reactions) == 0:
+            await ctx.send(":x: Die angegebene Nachricht besitzt keine Reaction-Roles.")
+            return
+
+        if self._db_connector.check_reaction_role_uniqueness(message.id):
```

Comment:
Maybe rename the method to `is_reaction_role_unique` or `is_reaction_role_exclusive` so it reads easier in the if statements and better reflects the type of response coming from the method (boolean)